client/src/app/components/Header.tsx

```diff
@@ -99,7 +94,9 @@ class Header extends Component<HeaderProps, HeaderState> {
                                 >
                                     <div className="label">Utilities</div>
                                     <div className="icon">
-                                        <DropdownIcon />
+                                        <span className="material-icons">
```

Comment:
The line height of the navbar elements (explorer, visualizer, utilities) now also create a bit of visual misalignment, see the utilities chevron
![image](https://user-images.githubusercontent.com/3624944/156155601-5869a7e7-797e-4abe-b71e-1dd2da0dfae3.png)

Reply:
![Screenshot from 2022-03-01 12-43-47](https://user-images.githubusercontent.com/72078237/156164829-a597a14c-251a-4584-a57b-26ca50967d36.png)

Resolved